02-express-tutorial/controlers/people.js

```diff
@@ -0,0 +1,59 @@
+let { people } = require("../data");
```

Comment:
Requires change:
Imports (`require` files) will always use `const` instead of `let`